servicebroker/lib/index.js

```diff
@@ -1,50 +1,8 @@
-'use strict';
-
-var express = require('express');
-var basicAuth = require('basic-auth');
-var bodyParser = require('body-parser');
-
-var fs = require('fs');
+'use strict'
 var path = require('path');
-var logger = require(path.join(__dirname, './logger/logger.js'));
-var settings = require(path.join(__dirname, './config/setting.js'))((JSON.parse(
-  fs.readFileSync(path.join(__dirname, '../config/settings.json'), 'utf8'))));
-var validateResult = settings.validate();
-if (validateResult.valid === false) {
-  logger.error("Invalid configuration: " + validateResult.message);
-  throw new Error('settings.json is invalid');
+var args = process.argv;
+if (args.length == 4 && args[2] == "-c" && args[3] != "") {
```

Comment:
Can we flip this so we throw the new Error if the args are wrong.
Then we just have the var server = require ... as the last line which makes it obvious.